modules/nw-ptp-configuring-linuxptp-services-as-boundary-clock.adoc

```diff
@@ -52,108 +54,107 @@ spec:
       #utc_offset                       37
       clockClass                        248
       clockAccuracy                     0xFE
-      offsetScaledLogVariance         0xFFFF
+      offsetScaledLogVariance           0xFFFF
       free_running                      0
-      freq_est_interval               1
+      freq_est_interval                 1
       dscp_event                        0
       dscp_general                      0
-      dataset_comparison              G.8275.x
-      G.8275.defaultDS.localPriority  128
+      dataset_comparison                G.8275.x
+      G.8275.defaultDS.localPriority    128
       #
       # Port Data Set
       #
-      logAnnounceInterval          -3
-      logSyncInterval                -4
-      logMinDelayReqInterval       -4
-      logMinPdelayReqInterval      -4
-      announceReceiptTimeout       3
-      syncReceiptTimeout           0
-      delayAsymmetry                 0
-      fault_reset_interval         4
-      neighborPropDelayThresh      20000000
-      masterOnly                     0
-      G.8275.portDS.localPriority  128
+      logAnnounceInterval              -3
+      logSyncInterval                  -4
+      logMinDelayReqInterval           -4
+      logMinPdelayReqInterval          -4
+      announceReceiptTimeout            3
+      syncReceiptTimeout                0
+      delayAsymmetry                    0
+      fault_reset_interval              4
+      neighborPropDelayThresh           20000000
+      masterOnly                        0
+      G.8275.portDS.localPriority       128
       #
-      # Run time options
+      # Runtime options
       #
-      assume_two_step              0
-      logging_level                6
-      path_trace_enabled         0
-      follow_up_info               0
-      hybrid_e2e                   0
-      inhibit_multicast_service  0
-      net_sync_monitor           0
-      tc_spanning_tree           0
-      tx_timestamp_timeout       10
-      #was 1 (default !)
-      unicast_listen          0
-      unicast_master_table  0
-      unicast_req_duration  3600
-      use_syslog              1
-      verbose                   0
-      summary_interval      -4
-      kernel_leap             1
-      check_fup_sync          0
+      assume_two_step                   0
+      logging_level                     6
+      path_trace_enabled                0
+      follow_up_info                    0
+      hybrid_e2e                        0
+      inhibit_multicast_service         0
+      net_sync_monitor                  0
+      tc_spanning_tree                  0
+      tx_timestamp_timeout              10
```

Comment:
Same note in this case?

_For Intel Columbiaville 800 Series NICs, set tx_timestamp_timeout to 50._